cgminer/Makefile

```diff
@@ -26,6 +26,10 @@ ifeq ($(CONFIG_CGMINER_AVALON7),y)
 	PKG_VERSION:=git-$(shell git ls-remote https://github.com/Canaan-Creative/cgminer master | cut -f1 | cut -c1-7)
 	PKG_REV:=master
 endif
+ifeq ($(CONFIG_CGMINER_AVALON8),y)
+	PKG_VERSION:=git-$(shell git ls-remote https://github.com/XzxEmbedded/cgminer avalon8 | cut -f1 | cut -c1-7)
```

Comment:
@XzxEmbedded Change to Canaan repo please. and also pull new request to Canaan' CGMiner.